app/webpacker/components/RegistrationsV2/Register/StripeWrapper.jsx

```diff
@@ -55,7 +51,7 @@ export default function StripeWrapper({
           options={{ amount: data?.api_amounts.stripe ?? initialAmount, currency: competitionInfo.currency_code.toLowerCase(), mode: 'payment' }}
         >
           <PaymentStep
-            handleDonation={handleDonation}
+            handleDonation={setDonationAmount}
```

Comment:
You renamed it to `setDonationAmount` in the function component props, but you still pass it as `handleDonation`...